### PR TITLE
4.0.0.pre6!

### PR DESCRIPTION
**Pull request by jasonrclark**
Long time coming, well overdue!

* [x] CHANGELOG updates
* [x] Version bump
* [x] Test samples (JRuby 9.1)
* [x] Test on JRuby 9.0
* [x] Move existing [pre6](https://github.com/shoes/shoes4/milestone/9) items over to [pre7](https://github.com/shoes/shoes4/milestone/10)
* [x] 💃

### CODE REVIEW

**Comment by PragTob** [comment]
One minor thing otherwise looks good to me :+1:

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,33 @@
+v4.0.0.pre6 (33 commits)
+------------------------------
+
+Improvements (4)
+----------------
+
+* Create borderless windows [f85f8f2]
+* Lighten the load on Travis a bit [9ab21f5]
```
Comment (jasonrclark) [inline CHANGELOG]: You're right... must have gotten tagged as an improvement in the commit message and showed up in the automated list from `rake changelog`. Fixed!

**Comment by jasonrclark** [comment]
Looks like I'll have a few cycles today to finish up my testing work on 9.0.5.0. Once that's done and we're all green across the board, we'll get this thing out the door (just in time for RubyConf!)

**Comment by jasonrclark** [comment]
Hmm, looks like things got unhappy with the settings changes on master not being on the branches? Trying a rebase to see if that happy-s things up.

**Comment by jasonrclark** [comment]
Will merge and release this tomorrow unless I hear otherwise from anyone. 👟 👞 👡 👠

**Comment by PragTob** [comment]
LGTM :)

**Comment by jasonrclark** [comment]
And it's away!

**Comment by PragTob** [comment]
celebrate

**Pull request merged**